ansible/docker_site.yml

```diff
@@ -37,9 +36,9 @@
                                                                  "benchmarking", "check_install"]}
     - {role: ci/update_dependencies, when: run_build, tags: ["build", "unit_tests", "software_tests", "all_tests",
                                                              "code_coverage", "codecov_tool", "python_code_coverage",
-                                                             "cpp_code_coverage", "check_deb_make",
+                                                             "cpp_code_coverage", "check_deb_make", "code_style_check",
                                                              "complete_deb_check", "check_build", "benchmarking",
-                                                             "check_install"]}
+                                                             "check_install", "code_style_check"]}
```

Comment:
Why is this getting called twice in `update_dependencies`?

Reply:
This one was a mistake, will change :)